app/modules/asset_groups/models.py

```diff
@@ -413,26 +411,25 @@ def detected(self, job_id, data):
 
             for annot_id in range(len(results[asset_id])):
                 annot_data = results[asset_id][annot_id]
-                uuid = annot_data.get('uuid', None)
+                annot_uuid = annot_data.get('uuid', None)
```

Comment:
I'm having trouble seeing what the problem was. What was going wrong with the old code?

Reply:
Isn't it because by assigning something to `uuid`, we're overwriting the module `uuid` from `import uuid` and so if afterwards we need to do something like `uuid.UUID` it'll cause an error because `uuid` is no longer the module?

Reply:
Exactly that Karen, comment added in previous block, 
Basically the lesson is don't call variables uuid :)